release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -27,10 +27,10 @@ You must use {op-system} machines for the control plane, and you can use either
 //TODO: Add this for 4.14
 Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
 
-Starting with {product-title} {product-version}, Extended Update Support (EUS) is extended to 64-bit ARM, {ibm-power-name} (ppc64le), and {ibm-z-name} (s390x) platforms. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to 64-bit ARM, {ibm-power-name} (ppc64le), and {ibm-z-name} (s390x) platforms. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
 
 //TODO: Add the line below for EUS releases.
```

Comment:
I'll ask an RN writer but maybe this comment could be clarified to say the following:
```suggestion
//TODO: Add the line below for EUS releases and comment it out for non-EUS releases.
```

Reply:
confirmed with @cbippley that this change is good to implement